guillotina/api/content.py

```diff
@@ -206,6 +207,8 @@ async def __call__(self):
             obj = await create_content_in_container(
                 self.context, type_, new_id, check_constraints=True, **options
             )
+        except NoPermissionToAdd as e:
+            raise HTTPUnauthorized(content={"text": e.__repr__()})
```

Comment:
I think its better to add this here: https://github.com/plone/guillotina/blob/caa0ba163ebc1bc913f8f47aa4171b90fead81a6/guillotina/exc_resp.py, like this handler:

https://github.com/plone/guillotina/blob/caa0ba163ebc1bc913f8f47aa4171b90fead81a6/guillotina/exc_resp.py#L83-L88

Reply:
okay! guillotina is too big to bear sometimes X)